server/TenantPipeline/lib/serverless-saas-stack.ts

```diff
@@ -104,23 +154,144 @@ export class ServerlessSaaSStack extends cdk.Stack {
 
     const deployOutput = new codepipeline.Artifact();
 
-
     //Add the Lambda function that will deploy the tenant stack in a multitenant way
     pipeline.addStage({
-      stageName: 'Deploy',
+      stageName: 'PrepDeploy',
```

Comment:
In TenantPipeline_V2 diagram, this PrepareDeployment stage is missing. Just want to know, if there is a specific reason to skip it.

Reply:
yes, just wanted to keep it simple in the diagram.